internal/services/redis/redis_cache_resource.go

```diff
@@ -553,6 +558,11 @@ func resourceRedisCacheCreate(d *pluginsdk.ResourceData, meta interface{}) error
 		parameters.Properties.SubnetId = utils.String(v.(string))
 	}
 
+	if v, ok := d.GetOk("access_keys_authentication_disabled"); ok {
```

Comment:
The second return value of `d.GetOK` indicates the value of the given key has been set to a non-zero value (e.g. `false` for boolean). So this only tells if this value is true (`ok`), or false (`!ok`). In case it is `false`, shall we also set it and send it to the API, instead of leaving it null?

Reply:
The `GetOk` doc says 

```
// GetOk returns the data for the given key and whether or not the key
// has been set to a non-zero value at some point.
//
// The first result will not necessarilly be nil if the value doesn't exist.
// The second result should be checked to determine this information.
```

I believe if we get `false` value from configuration we'll get `v` = `false` and `ok` = `true` and the value will be sent to the API. I'll give this a test.